allennlp/tests/commands/find_learning_rate_test.py

```diff
@@ -178,3 +180,82 @@ def test_search_learning_rate_without_stopping_factor(self):
         learning_rates, losses = search_learning_rate(self.trainer, num_batches=100, stopping_factor=None)
         assert len(learning_rates) == 101
         assert len(losses) == 101
+
+class TestFindLearningRateForMultiGPU(AllenNlpTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.params = lambda: Params({
+                        "model": {
+                            "type": "simple_tagger",
+                            "text_field_embedder": {
+                                "tokens": {
+                                    "type": "embedding",
+                                    "embedding_dim": 5
+                                }
+                            },
+                            "encoder": {
+                                "type": "lstm",
+                                "input_size": 5,
+                                "hidden_size": 7,
+                                "num_layers": 2
+                            }
+                        },
+                        "dataset_reader": {"type": "sequence_tagging"},
+                        "train_data_path": str(self.FIXTURES_ROOT / 'data' / 'sequence_tagging.tsv'),
+                        "validation_data_path": str(self.FIXTURES_ROOT / 'data' / 'sequence_tagging.tsv'),
+                        "iterator": {"type": "basic", "batch_size": 2},
+                        "trainer": {
+                            "cuda_device": [0, 1],
+                            "num_epochs": 2,
+                            "optimizer": "adam"
+                        }
+                    })
+
+    @pytest.mark.skipif(torch.cuda.device_count() < 2,
+                        reason="Need multiple GPUs.")
+    def test_find_learning_rate(self):
+        find_learning_rate_model(self.params(), os.path.join(self.TEST_DIR, 'test_find_learning_rate'),
+                                 start_lr=1e-5,
+                                 end_lr=1,
+                                 num_batches=100,
+                                 linear_steps=True,
+                                 stopping_factor=None,
+                                 force=False)
+
+        # It's OK if serialization dir exists but is empty:
```

Comment:
You don't need anything below this line; it's testing stuff that we already test elsewhere that has nothing to do with multi-GPU operation.  The call to `find_learning_rate_model` above is sufficient.